design/sketch.webidl

```diff
@@ -195,18 +195,16 @@ interface WebGPUPipelineLayout {
 };
 
 // BindGroup
-dictionary WebGPUBufferBinding {
-    WebGPUBuffer buffer;
-    u32 offset;
-    u32 size;
-};
+dictionary WebGPUBinding {
+    uint32_t binding;
 
-typedef (WebGPUSampler or WebGPUTextureView or WebGPUBufferBinding) WebGPUBindingResource;
+    WebGPUSampler? sampler;
```

Comment:
Optionals in dictionaries don't make much sense

Reply:
I don't see why optionals in dictionaries don't make sense. Is it because all dictionary members are always optional? The WebIDL checker seems ok with them.

Reply:
Yep.